cluster/gce/util.sh

```diff
@@ -744,6 +744,17 @@ function check-cluster() {
       local elapsed=$(($(date +%s) - ${start_time}))
       if [[ ${elapsed} -gt ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} ]]; then
           echo -e "${color_red}Cluster failed to initialize within ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} seconds.${color_norm}" >&2
+          if [[ ${KUBE_TEST_DEBUG-} =~ ^[yY]$ ]]; then
+            local tmp_log="$(mktemp)"
+            local file
+            for file in /var/log/startupscript.log /var/log/kube-apiserver.log; do
+              echo "${file} contents:"
+              if gcloud compute copy-files "${MASTER_NAME}:${file}" "${tmp_log}"; then
+                cat "${tmp_log}"
```

Comment:
an alternative to `cat`ing these files: if `E2E_REPORT_DIR` is set (which it is on Jenkins runs), save these logs there; they are automatically uploaded to GCS.